cmd/ingester/app/consumer/consumer_metrics.go

```diff
@@ -30,8 +30,17 @@ type errMetrics struct {
 	errCounter metrics.Counter
 }
 
+type partitionMetrics struct {
```

Comment:
nit: we should clean-up these metrics. All 3 structs are per-partition, could we not combine them into one? Then we won't have all these small functions on the Consumer.

Reply:
I agree - I'll do it as a separate commit